new-pipeline/src/dist.js

```diff
@@ -1163,12 +1168,12 @@ function displaySingleHistogramSet(axes, useTable, histograms, title,
       max_y: maxPercentage + 2, // Add some extra space to account for the top label
       transition_on_update: false,
       target: axes,
-      x_label: histogram.description,
```

Comment:
You could do what I did with the grouped barchart and put a bottom of 150 on ones that need it.